app/pods/publications/publication/documents/controller.js

```diff
@@ -144,10 +147,9 @@ export default class PublicationDocumentsController extends Controller {
   *savePiece(piece) {
     const documentContainer = yield piece.documentContainer;
     yield documentContainer.save();
+    this.model.pushObject(piece);
+    piece.cases.pushObject(this.case);
```

Comment:
Misschien is het hier beter dat ik, naar analogie met de filter, het model reload?